app/app.go

```diff
@@ -520,7 +520,9 @@ func NewApp(
 
 	evmBankKeeper := evmutilkeeper.NewEvmBankKeeper(app.evmutilKeeper, app.bankKeeper, app.accountKeeper)
 	app.evmKeeper = evmkeeper.NewKeeper(
-		appCodec, keys[evmtypes.StoreKey], tkeys[evmtypes.TransientKey],
+		appCodec, legacyAmino,
+		keys[evmtypes.StoreKey], tkeys[evmtypes.TransientKey],
+		keys[paramstypes.StoreKey], tkeys[paramstypes.TStoreKey],
```

Comment:
can we test in the migration or e2e tests that these keys get set?

Reply:
The e2e upgrade test actually covers this 🚀